server/integration_tests/test_data/detection_api_multivar/numberofpoorhispanicwomenwithphd/debug_info.json

```diff
@@ -167,46 +162,45 @@
     },
     "Query": "number of poor hispanic women with phd",
     "SV": [
-      "dc/06f0jf8xvzw4f",
```

Comment:
this is a really relevant sv: https://datacommons.org/browser/dc/06f0jf8xvzw4f

Reply:
Looks like dc/06f0jf8xvzw4f just got demoted to second highest, not deleted. It's at line 166 now.